### PR TITLE
[cf]-data-exporter-docs-update

### PR DESCRIPTION
**Pull request by colefreeman**
# Description
This pull request updates the Data Exporter documentation 

# How Has This Been Tested?
The doc is showing as expected in mintlify dev env.

- [ ] Test A
- [ ] Test B

# Checklist
- [X] The PR is tagged with proper labels (bug, enhancement, feature, documentation)
- [X] I have performed a self-review of my own code
- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation

cc:
@johnson-mage 

Screenshot 2024-07-02 at 11 08 24 AM

Screenshot 2024-07-02 at 11 10 02 AM

Screenshot 2024-07-02 at 11 13 21 AM

Screenshot 2024-07-02 at 11 14 24 AM

Screenshot 2024-07-02 at 11 15 08 AM

Screenshot 2024-07-02 at 11 15 54 AM

Screenshot 2024-07-02 at 11 16 27 AM

Screenshot 2024-07-02 at 11 17 33 AM

Screenshot 2024-07-02 at 11 18 16 AM

Screenshot 2024-07-02 at 11 18 43 AM

### CODE REVIEW

File: docs/design/blocks/data-exporter.mdx [hunk 1]
```diff
@@ -1,47 +1,274 @@
...
-  "Once you’re done transforming your data, write code in these types of blocks to
-  store that data or to train models and store those models elsewhere."
+  "After completing data transformations, utilize the data exporter blocks to either load the processed 
+  data or store a machine learning model in an external data storage system."
 ---
 
-## Example
+<img
+  alt="exporter"
+  src="https://github.com/mage-ai/assets/blob/main/blocks/exporter/world-export.gif?raw=True"
```
Comment (johnson-mage) [inline docs/design/blocks/data-exporter.mdx:10]: @colefreeman Can we change this gif to something without a company's logo? Just seems odd to have one with "eraplast electric" on it. And then ill approve the PR

Comment (colefreeman) [reply docs/design/blocks/data-exporter.mdx]: Fixed

**Comment by johnson-mage** [comment]
@colefreeman Can we clean up this [folder](https://github.com/mage-ai/assets/tree/main/blocks/exporter) in the assets repo? Rename `image (2).gif` to something more descriptive, and remove `test.txt` and `world-export.gif` if we are not using those anymore.

File: docs/design/blocks/data-exporter.mdx (hunk 1 above)
Comment (colefreeman) [reply docs/design/blocks/data-exporter.mdx]: @johnson-mage deleted the files and made the change for the asset tag in the doc and it is appearing in mintlify dev env

**Review by johnson-mage — APPROVED** [review]

**Pull request merged**